doc/architecture/database_schemata.md

```diff
@@ -149,3 +202,57 @@ subfamily }|--|| family : ""
 | **is_tree**                      | true                             | Set of Herbaceous/Woody (woody) AND life cycle (perennial)                                        |
 | **nutrition_demand**             | NULL                             | If "Nutritionally poor soil" in `environmental_tolerances` is given `light feeder` should be set. |
 | **preferable_permaculture_zone** | NULL                             |                                                                                                   | -1..6 (-1 should be printed as 00) |
+
+## `Maps`
+
+| **_Column name_**                | **_Example_**                    | **_Initial rule_**                                                                                | **_Description_**                  |
+| :------------------------------- | :------------------------------- | :------------------------------------------------------------------------------------------------ | :--------------------------------- |
+| **id**                           | 1                                |
+| **owner_id**                     | 1                                |
+| **name**                         | My Map                           |
+| **is_variant**                   | false                            |
+| **version_name**                 | NULL                             |
+| **is_inactive**                  | false                            |
+| **last_visit**                   | 2023-04-04                       |
+| **honors**                       | 0                                |
```

Comment:
What are the constraints for this value? 
Can it grow up to infinity? 
Can it be negative? 

If this information is already present in some other doc, please just add a link to prevent information from going out of sync.